### ISSUE TITLE
Open_Uri Redirection Error

### ISSUE THREAD

**Issue by n-qw** [HuggableSquare/vsco-dl#2]
This error began happening a few days ago, and it affects every account, including those successfully ripped in the past, which I take to mean there has been a change on the VSCO website. After a dozen or so images have been downloaded, the following error is printed and the script won't download beyond that point. Re-running the script has no effect, it always errors at the same point, which varies from account to account.

The ~ signs are not what is actually printed, they are just placeholders for any image url.

```
Ruby24-x64/lib/ruby/2.4.0/open-uri.rb:225:in `open_loop': redirection forbidden: https://im.vsco.co/aws-us-west-2/~/~/~/~.jpg -> http://image-aws-us-west-2.vsco.co/~/~/~/~.jpg (RuntimeError)
        from Ruby24-x64/lib/ruby/2.4.0/open-uri.rb:151:in `open_uri'
        from Ruby24-x64/lib/ruby/2.4.0/open-uri.rb:721:in `open'
        from Ruby24-x64/lib/ruby/2.4.0/open-uri.rb:35:in `open'
        from vsco-dl.rb:82:in `block in <main>'
        from vsco-dl.rb:65:in `each'
        from vsco-dl.rb:65:in `each_with_index'
        from vsco-dl.rb:65:in `<main>'
```

I tried fixing this by installing an open_uri_redirections gem. This did not fix the error.

**Comment by HuggableSquare** [comment]
This was a weird bug, vsco's endpoint occasionally does a redirect to an http image instead of https, and I really can't figure out _why_. Changing referer occasionally fixes it for some images, but then breaks it again for others, so I decided to just request all images using http instead of https. This isn't ideal, but it shouldn't be a big issue.

tl;dr: I pushed up a fix and it should work now. If you continue to have issues or have any questions don't hesitate to post them here. I hope my little project continues to be useful for you! 😄
